File: Baekjoon/11723.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
const [m, ...dataes] = fs.readFileSync(filePath).toString().trim().split("\n");
let groupS = new Map();

dataes.forEach((ele) => {
  let [calculation, num] = ele.split(" ");
  if (calculation === "add") {
    if (!groupS.has(num)) groupS.set(num);
  } else if (calculation === "remove") {
    if (groupS.has(num)) groupS.delete(num);
  } else if (calculation === "check") {
    groupS.has(num) ? console.log(1) : console.log(0);
  } else if (calculation === "toggle") {
    groupS.has(num) ? groupS.delete(num) : groupS.set(num);
  } else if (calculation == "all") {
    for (i = 1; i <= 20; i++) {
      groupS.set(i.toString());
    }
  } else groupS.clear();
});

// 언어 제한으로 인해 node.js로 풀면 컴파일 에러 발생

File: python/Baekjoon/11723.py
import sys

m = int(sys.stdin.readline().rstrip())
s = set()
fullS = set(range(1, 21))

for _ in range(m):
    command = sys.stdin.readline().rstrip().split()
    if command[0] == "all": s = fullS.copy()
    elif command[0] == "empty": s.clear()
    else:
        comm, x = command[0], int(command[1])

        if comm == "add": s.add(x)
        elif comm == "remove": s.discard(x)
        elif comm == "toggle": s.remove(x) if x in s else s.add(x)
        elif comm == "check": print("1" if x in s else "0")
